src/multiparser.js

```diff
@@ -42,8 +42,9 @@ function fromBabylonFlowOrTypeScript(path) {
   switch (node.type) {
     case "TemplateLiteral": {
       const isCss = [isStyledJsx, isStyledComponents].some(isIt => isIt(path));
+      const isEmpty = node.quasis.length === 1 && !node.quasis[0].value.raw;
```

Comment:
Actually, I think we should add `.trim()` so that ``styled.div`  ` `` is formatted to ```styled.div`` ``` and not broken across several lines.

Reply:
@lydell wouldn't that change the AST?

Reply:
No, why would it?

Reply:
It's changing the contents of the literal; the value of the first quasis changes from `" "` to `""`

Reply:
(However, technically, adding newlines also changes the value of the literal. It just happens to not matter for styled-components. So changing it here might be fine)

Reply:
Yes, since we know that the contents is CSS we are free to change it however we want – which we also already do.

Reply:
You're right, that's fair. That means the solution I proposed won't work, since it skips the CSS formatting at all, leaving it up to the default printer.